appveyor.yml

```diff
@@ -56,34 +63,50 @@ matrix:
 
 init:
 # Replaces the placeholder version with a custom format: BUILD_NUMBER-BRANCH-SHORT_GIT_SHA
-  - ps: Update-AppveyorBuild -Version "$env:appveyor_build_number-$env:appveyor_repo_branch-$($env:appveyor_repo_commit.substring(0,7))"
-  - systeminfo
+  - ps: |
+      if ($isWindows) {
+        Update-AppveyorBuild -Version "$env:appveyor_build_number-$env:appveyor_repo_branch-$($env:appveyor_repo_commit.substring(0,7))"
+        iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/ab48d6e110f2feb585b6b908a72d6e7dbfb83cb3/scripts/enable-rdp.ps1'))
+        systeminfo
+      }
 # Attempt to ensure we don't try to convert line endings to Win32 CRLF as this will cause build to fail
-  - git config --global core.autocrlf true
-# Allows RDP, uses Git tag build-agent-v6.1.0%2B1300
-  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/ab48d6e110f2feb585b6b908a72d6e7dbfb83cb3/scripts/enable-rdp.ps1'))
-  - echo "%APPVEYOR_REPO_COMMIT%">"%TMP%\\REVISION_appveyor_%ERTS_VERSION%"
+  - cmd: git config --global core.autocrlf true
+  - sh: git config --global core.autocrlf input
+  - sh: curl -sflL 'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-ssh.sh' | bash -e -
+  - cmd: echo "%APPVEYOR_REPO_COMMIT%">"%TMP%\\REVISION_appveyor_%ERTS_VERSION%"
 
 install:
-  - '%APPVEYOR_BUILD_FOLDER%\scripts\windows\msys2_prepare.bat'
+  - cmd: '%APPVEYOR_BUILD_FOLDER%\scripts\windows\msys2_prepare.bat'
+  - sh: '$APPVEYOR_BUILD_FOLDER/ci/appveyor/linux_prepare.sh'
 
 build_script:
-  - '%APPVEYOR_BUILD_FOLDER%\ci\appveyor\build.bat'
+  - cmd: '%APPVEYOR_BUILD_FOLDER%\ci\appveyor\build.bat'
+  - sh: make prod-build
 
 after_build:
-  - '%APPVEYOR_BUILD_FOLDER%\ci\appveyor\package.bat'
+  - cmd: '%APPVEYOR_BUILD_FOLDER%\ci\appveyor\package.bat'
 
-# test_script:
-#  - '%APPVEYOR_BUILD_FOLDER%\ci\appveyor\test.bat'
+test_script:
+#  - cmd: '%APPVEYOR_BUILD_FOLDER%\ci\appveyor\test.bat'
+  - sh: epmd -daemon
+  - sh: make ct
 
 deploy: off
 
 on_finish:
 # Set blockRdp to true to allow RDP, uses Git tag build-agent-v6.1.0%2B1300
-  - ps: $blockRdp = $false; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/ab48d6e110f2feb585b6b908a72d6e7dbfb83cb3/scripts/enable-rdp.ps1'))
+  - ps: |
+      if ($isWindows) {
+        $blockRdp = $false; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/ab48d6e110f2feb585b6b908a72d6e7dbfb83cb3/scripts/enable-rdp.ps1'))
+      }
+# Set APPVEYOR_SSH_BLOCK to true to allow SSH access
+  - sh: export APPVEYOR_SSH_BLOCK=false
+  - sh: curl -sflL 'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-ssh.sh' | bash -e -
```

Comment:
Is this run above already?

Reply:
What exactly do you mean?

Reply:
At line 75 I see same `   - sh: curl -sflL 'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-ssh.sh' | bash -e - `

Reply:
It is required because the previously set var `APPVEYOR_SSH_BLOCK` will make the script behave differently than during the init phase. Same with the rdp script.